### ISSUE TITLE
ERROR: vue-loader was used without the corresponding plugin. Make sure to include VueLoaderPlugin in your webpack config.

### ISSUE THREAD

**Issue by anto-christo** [madlabsinc/mevn-boilerplate#81]
Hi @jamesgeorge007 @ajomadlabs , I'm one of the mentors from GSSoC assigned to mevn-cli. I was trying to run the cli and faced an issue. The issues was not with the CLI but with this boilerplate (and maybe with other boilerplates as well).

After running `mevn run:client`, the following error message was thrown.

```
> client@1.0.0 dev /home/anto/Code/test-app/client
> webpack-dev-server --inline --progress --config build/webpack.dev.conf.js

 10% building modules 3/4 modules 1 active ...client/index.js?http://localhost:8080(node:16262) [ESLINT_LEGACY_OBJECT_REST_SPREAD] DeprecationWarning: The 'parserOptions.ecmaFeatures.experimentalObjectRestSpread' option is deprecated. Use 'parserOptions.ecmaVersion' instead. (found in "standard")
 94% asset optimization ERROR  Failed to compile with 5 errors15:53:55               

vue-loader was used without the corresponding plugin. Make sure to include VueLoaderPlugin in your webpack config.

You may use special comments to disable some warnings.
Use // eslint-disable-next-line to ignore the next line.
Use /* eslint-disable */ to ignore all warnings in a file.
 error  in ./src/App.vue?vue&type=style&index=0&lang=css&

Module parse failed: Unexpected character '#' (15:0)
You may need an appropriate loader to handle this file type.
| 
| 
| #app {
|   font-family: 'Avenir', Helvetica, Arial, sans-serif;
|   -webkit-font-smoothing: antialiased;

 @ ./src/App.vue 4:0-63
 @ ./src/main.js
 @ multi (webpack)-dev-server/client?http://localhost:8080 webpack/hot/dev-server ./src/main.js
```

I was able to fix it by importing VueLoaderPlugin from vue-loader and adding it's object to the plugins array in webpack config. We have to do this since certain changes were made in v15 of vue-loader.

Is this issue being faced by others as well or is just my environment causing the issue? And should I raise a PR for this?

**Comment by ajomadlabs** [comment]
@anto-christo You could go ahead and raise a PR for this.

**Comment by jamesgeorge007** [comment]
Fixed in #88
